scripts/filemanager.js

```diff
@@ -9,8 +9,34 @@
  *	@author		Pavel Solomienko <https://github.com/servocoder/>
  *	@copyright	Authors
  */
-
+ 
 (function($) {
+  
+  // keep variables in non global object
+var fm = {};
```

Comment:
I believe this object is redundant.
All variables defined inside `(function($) { })` exist in the scope of that function and they aren't accessible in the global scope. Vice versa - variables defined inside `(function($) { })` aren't going to overwrite global variables for some other scope. You could simply check it if you invoke `config` or `lg` variable in a debug console of your browser FM is inited. They are `undefined`, in case you don't have globals defined with the same name of course.

So I can't see any benefit from the `fm` object. Only the headache to add it for the each variable defined in the FM scope. Update your PR and remove it, if you don't have any reasonable objections, please.